.gitignore

```diff
@@ -11,3 +11,4 @@
 .vagrant/
 doc/build/
 htmlcov/
+*.sw[po]
```

Comment:
If the developer uses vim, then they should be ignoring these in their own global gitignore file.  The project shouldn't be aware of this.

Your `~/.gitconfig` can have a section pointing to a global .gitignore. 

```
[core]
	excludesfile = /Users/jodewey/.gitignore
	
```